gcc/rust/checks/errors/borrowck/rust-bir-place.h

```diff
@@ -255,7 +256,9 @@ class PlaceDB
 
   PlaceId add_variable (NodeId id, TyTy::BaseType *tyty)
   {
-    return add_place ({Place::VARIABLE, id, {}, is_type_copy (tyty), tyty}, 0);
+    return add_place (
+      {Place::VARIABLE, id, {}, is_type_copy (tyty), tyty, tyty->get_locus ()},
```

Comment:
Using location of type does not seem right. As I mentioned before, we might not oven need it here. Alternative could be to extract the location from NodeId. But that can always be done on demand.

Reply:
actually I was first trying to use get_ast_item() method to retrieve the location, but it seems like the location is copied from AST->HIR->TyTy, so getting locus from NodeId or the TyTy should be same, unless we plan to change the location information of TyTy in future